.travis.yml

```diff
@@ -6,5 +6,5 @@ cache: bundler
 before_script: cp config/database.yml.sample config/database.yml
 language: ruby
 rvm:
-  - 2.1.5
+  - 2.3.3
```

Comment:
Hm, I think apps should just pick one version and stick with it, unless you are targeting the MRI versions shipped with a distro...